queries.sql

```diff
@@ -23,3 +23,57 @@ select * from animals where animal_name <> 'Gabumon';
 
 /*All animals with a weight between 10.4kg and 17.3kg*/
 select * from animals where weight_kg between 10.4 and 17.3;
+
+
+/* Transactions 1: Set species to 'unspecified' and Rollback*/
+begin;
+update animals set species = 'unspecified';
+
+rollback;
+
+/*Transaction 2: Set species based on name and commit*/
+begin;
+update animals set species = 'digimon' where animal_name like '%mon';
+update animals set species = 'pokemon' where species is null;
+commit;
```

Comment:
Great job so far. It would be great to verify that the changes were made in the table before and after `commit` the transaction. You just need to add `SELECT species from animals` before and after `commit`. So that you can verify whether the change was made or not and is undone. Kindly, add it.

![Screenshot from 2023-09-26 15-06-08](https://github.com/iankiema/Vet-Clinic-Database/assets/101523582/42144e4d-0f66-4a13-a57c-03e8bbcbb651)